tests/routes/test_peers.py

```
@@ -561,8 +562,72 @@ def test_search_peer_nonexistent(
+    assert response.status_code == 200
+    data = response.json()
+
+    # Response should have pagination structure
+    assert "items" in data
+    assert "total" in data
+    assert "page" in data
+    assert "size" in data
+    assert isinstance(data["items"], list)
+
+
+def test_search_peer_with_semantic_search_true_disabled(
```

Ruff (0.11.9)

Missing return type annotation for public function `test_search_peer_with_semantic_search_true_disabled`

Add return type annotation: `None`

(ANN201)